uw_bridge/custom_fields.py

```diff
@@ -18,19 +18,19 @@ class CustomFields:
     def __init__(self):
         self.fields = []
         self.name_id_map = {}
+        self.id_name_map = {}
         self.get_custom_fields()
 
     def get_custom_fields(self):
         resp = get_resource(URL)
         resp_data = json.loads(resp)
-        if resp_data.get("custom_fields") is not None:
-            for field in resp_data["custom_fields"]:
-                if (field.get("id") is not None and
-                        field.get("name") is not None):
-                    cf = BridgeCustomField(field_id=field["id"],
-                                           name=field["name"].lower())
-                    self.fields.append(cf)
-                    self.name_id_map[cf.name] = cf.field_id
+        for field in resp_data["custom_fields"]:
```

Comment:
Can resp_data["custom_fields"] still be None?

Reply:
Not a legit case.